scss/utilities/_position.scss

```diff
@@ -23,3 +23,8 @@
     z-index: $zindex-sticky;
   }
 }
+
+.pos-s { position: static; }
```

Comment:
You're missing `position: sticky`, which, with this naming scheme, would be problematic. We'll need that utility and a different name.